ci-operator/step-registry/openshift-qe/orion/cluster-density/openshift-qe-orion-cluster-density-ref.yaml

```diff
@@ -11,9 +11,17 @@ ref:
     documentation: |-
       Override the e2e version
   - name: TAG
-    default: "v0.0.1"
+    default: "v0.0.2"
     documentation: |-
       Override branch of orion
+  - name: ES_BENCHMARK_INDEX
+    default: "ripsaw-kube-burner-*"
```

Comment:
Here, since we have the es_type set to internal as the default I think we should have ospst-perf-scale-ci-* and ospst-ripsaw-kube-burner* be the dfeaults for the metadata indexes. Thoughts?

Reply:
oh... yeah, maybe I need to swap that.